mod/search.php

```diff
@@ -214,7 +214,7 @@ function search_content(App $a) {
 			FROM `item` %s
 			WHERE %s AND (`item`.`uid` = 0 OR (`item`.`uid` = %s AND NOT `item`.`global`))
 				$sql_extra
-			GROUP BY `item`.`uri` ORDER BY `item`.`id` DESC LIMIT %d , %d",
+			GROUP BY `item`.`uri`, `item`.`id` ORDER BY `item`.`id` DESC LIMIT %d , %d",
```

Comment:
If we do not GROUP BY a unique key such as item.id, then we must use MIN() on all the SELECT fields. Furthermore, item.id is already selected twice in https://github.com/friendica/friendica/blob/7b352f3f74b045690cdeae507b609f7a9cc8db03/include/conversation.php#L421-L430 :-)